### ISSUE TITLE
Python3.9.2_32bit cannot load pytransform.pyd: Dll load failed (The specified procedure could not be found)

### ISSUE THREAD

**Issue by leouvarov** [dashingsoft/pyarmor#489]
Obfuscated (--advanced 2) script fails to run with following error:
ImportError: DLL load failed while importing pytransform: The specified procedure could not be found.

Environment:
64 bit Windows10 and 32 bit Python3.9.2 (the bug cannot be reproduced on Python3.9.2 64 bit)

```
(env39_32) C:\Users\leo\obfuscate_todel>pip install pyarmor
Collecting pyarmor
  Using cached pyarmor-6.6.2-py2.py3-none-any.whl (2.3 MB)
Installing collected packages: pyarmor
Successfully installed pyarmor-6.6.2
WARNING: You are using pip version 20.2.3; however, version 21.0.1 is available.
You should consider upgrading via the 'c:\users\leo\obfuscate_todel\env39_32\scripts\python.exe -m pip install --upgrade pip' command.

(env39_32) C:\Users\leo\obfuscate_todel>more test.py
print('Hi')

(env39_32) C:\Users\leo\obfuscate_todel>pyarmor obfuscate --advanced 2 test.py
INFO     PyArmor Trial Version 6.6.2
INFO     Target platforms: Native
INFO     Update target platforms to: ['windows.x86.11.py39']
INFO     Source path is "C:\Users\leo\obfuscate_todel"
INFO     Entry scripts are ['test.py']
INFO     Use cached capsule C:\Users\leo\.pyarmor\.pyarmor_capsule.zip
INFO     Search scripts mode: Normal
INFO     Save obfuscated scripts to "dist"
INFO     Read product key from capsule
INFO     Obfuscate module mode is 2
INFO     Obfuscate code mode is 1
INFO     Wrap mode is 1
INFO     Restrict mode is 1
INFO     Advanced value is 2
INFO     Super mode is True
INFO     Generating super runtime library to "dist"
INFO     Extract pytransform.key
INFO     Generate default license file
INFO     Copying C:\Users\leo\.pyarmor\platforms\windows\x86\11\py39\pytransform.pyd
INFO     Patch extension dist\pytransform.pyd
INFO     Patch library file OK
INFO     Generate runtime files OK
INFO     Use protection template: env39_32\lib\site-packages\pyarmor\protect_code2.pt
INFO     Start obfuscating the scripts...
INFO            test.py -> dist\test.py
INFO     Obfuscate 1 scripts OK.

(env39_32) C:\Users\leo\obfuscate_todel>cd dist

(env39_32) C:\Users\leo\obfuscate_todel\dist>python test.py
Traceback (most recent call last):
  File "C:\Users\leo\obfuscate_todel\dist\test.py", line 1, in <module>
    from pytransform import pyarmor
ImportError: DLL load failed while importing pytransform: The specified procedure could not be found.

```

**Comment by leouvarov** [comment]
I also performed the above steps on Windows10 32 bit and Python3.9.2 - and it failed the same way.

**Comment by jondy** [comment]
Does it work to import the original library directly by python39-32?
```
cd C:\Users\leo\.pyarmor\platforms\windows\x86\11\py39\
python39-32 -c "import pytransform"
```

**Comment by leouvarov** [comment]
No, it doesn't work either:
```
(env39_32) C:\Users\leo\.pyarmor\platforms\windows\x86\11\py39>dir
...
22/03/2021  17:23         1,225,230 pytransform.pyd
...
(env39_32) C:\Users\leo\.pyarmor\platforms\windows\x86\11\py39>python -c "import pytransform"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: DLL load failed while importing pytransform: The specified procedure could not be found.
```

**Comment by leouvarov** [comment]
and sha256sum of pytransform.pyd is
a52354a1fc727908c284febc8f99d86f4264fc11fdb032e917ee9ec36a413215

**Comment by jondy** [comment]
Could you check the dependency of this file by `dumpbin`?
```
cd C:\Users\leo\.pyarmor\platforms\windows\x86\11\py39
dumpbin /dependents pytransform.pyd
```

**Comment by leouvarov** [comment]
I don't have dumpbin on my system, but I used https://github.com/lucasg/Dependencies to produce dependencies list

Output of Dependencies.exe -chain ..\pytransformd.pyd is attached, let me know if this in not that you want
[Dependencies_chain_pytransform.txt](https://github.com/dashingsoft/pyarmor/files/6190116/Dependencies_chain_pytransform.txt)
.

**Comment by leouvarov** [comment]
Update: I ran gui version of Dependencies and it showed that api-ms-win-core-string-obsolete-l1-1-0.dll is cannot be found

**Comment by jondy** [comment]
Please first fix missing `api-ms-win-core-string-obsolete-l1-1-0.dll` issue by google or any other way, it's Windows system file.

**Comment by leouvarov** [comment]
I those errors have nothing to do with the issue, the message is about python39.dll, not pytransformer.pyd.

I think I found something interesting: I compared function names in pytransformer.pyd and python39.dll and they have different prefixes (_Py vs Py)

for instance PyAsyncGen_Type function. In pytransformer.pyd it's called _PyAsyncGen_Type (with underscore in the beginning)

As a result, this function cannot be found and import fails...
I made a check for python38_32 and names between pytransformer and python38.dd
![py38_vs_py39_function_names_dont_match](https://user-images.githubusercontent.com/47031166/112320651-438f3400-8cc0-11eb-89b4-24b6cb293147.jpg)
 do match. Screenshot (Dependencies pytransform.pyd 39 vs 39 is attached)

**Comment by jondy** [comment]
I have rebuild it to remove the leading underscore, please download it
https://pyarmor.dashingsoft.com/downloads/temp/pytransform.pyd

Then obfuscate python scripts for py39-32 by this way
```
pyarmor obfuscate --advanced 2 --platform \path\to\pytransform.pyd test.py
```

**Comment by leouvarov** [comment]
Jondy, please check the link, it's dead (404)

**Comment by jondy** [comment]
Sorry, try it again.

**Comment by leouvarov** [comment]
Works fine on Windows10 Home 32-bit
![rebuilt_dll_test_w10home32](https://user-images.githubusercontent.com/47031166/112395812-25512480-8d10-11eb-8010-7d0a19df1c90.jpg)

**Comment by jondy** [comment]
Please keep it, after v6.6.2, the trial version could not download extra platform's core libraries.

**Comment by jondy** [comment]
Please keep it, after v6.6.2, the trial version could not download extra platform's core libraries.

**Comment by jondy** [comment]
Fixed in v6.7.0 for both trial and purchased version.
